keras-runner/src/main/kotlin/org/jetbrains/research/deepbugs/keras/runner/deserializer/json/LayerConfig.kt

```diff
@@ -0,0 +1,90 @@
+package org.jetbrains.research.deepbugs.keras.runner.deserializer.json
+
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.nullable
+import kotlinx.serialization.json.JsonObject
+
+sealed class LayerConfig {
+    abstract var name: String
+    abstract var trainable: Boolean
+    abstract var batchInputShape: List<Int?>?
+    abstract var dtype: String?
+
+    @Serializable
+    data class Dense(
+        override var name: String,
+        override var trainable: Boolean,
+        @SerialName("batch_input_shape") override var batchInputShape: List<Int?>?,
+        override var dtype: String?,
+        var units: Int,
+        var activation: ActivationConfig,
+        @SerialName("use_bias") var useBias: Boolean
+    ) : LayerConfig() {
+        @Serializer(forClass = Dense::class)
+        companion object : KSerializer<Dense> {
+            override fun serialize(encoder: Encoder, obj: Dense) = Unit
+
+            override fun deserialize(decoder: Decoder): Dense {
```

Comment:
Look like it can be replaced with `@SerialName(...)` on class and change `classDiscriminator = "class"` in JsonConfiguration